src/components/organisms/ActionsPane/Diff/Diff.tsx

```diff
@@ -17,14 +17,14 @@ const Diff: React.FC<IProps> = props => {
 
   return (
     <Tooltip mouseEnterDelay={TOOLTIP_DELAY} title={tooltipTitle} placement="bottomLeft">
-      <PrimaryButton
+      <LinkButton
```

Comment:
Kind of confusing to be able to give a link button the type `primary`, it should only have the type of link

Reply:
Thanks for the comment mate, I need a new button with no borders what should be the name? I used LinkButton since I saw it is used for such components before but open to alternative names.

Reply:
Can we directly use the antd button with type link without needing to create an alternate custom component?